packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -50,10 +85,13 @@ class UserSessionManager implements IUserSessionManager {
 
     if (!persistedAnonymousId) {
       // TODO: implement the storage.getAnonymousId autoCapture functionality as plugin that takes options in
-      const autoCapturedAnonymousId = [undefined];
+      const autoCapturedAnonymousId = defaultPluginManager.invoke<string | undefined>(
```

Comment:
On Ln 73, is the `userSession.anonymousIdGoogleLinker` plugin defined?

Reply:
I believe George is working on it.

Reply:
Add a TODO please

Reply:
Sure

Reply:
Done